src/pymor/bindings/dunegdt.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2016 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+from pymor.core.config import config
+
+
+if config.HAVE_DUNEGDT:
+
+    from collections import OrderedDict
+    import os
+    import subprocess
+    import tempfile
+    from tempfile import mkstemp
+
+    from dune.xt.common.vtk.plot import plot as k3d_plot
+    from dune.gdt import DiscreteFunction
+
+    from pymor.core.base import ImmutableObject
+    from pymor.bindings.dunext import DuneXTVector
+    from pymor.vectorarrays.list import ListVectorArray
+
+
+    class DuneGDTParaviewVisualizer(ImmutableObject):
+        """Visualize a dune-gdt discrete function using paraview.
+
+        Parameters
+        ----------
+        space
+            The dune-gdt space for which we want to visualize DOF vectors.
+        """
+
+        def __init__(self, space):
+            self.__auto_init(locals())
+
+        def visualize(self, U, m, title=None, legend=None, separate_colorbars=False,
+                      rescale_colorbars=False, block=None, filename=None, columns=2):
+
+            def visualize_single(vec, vec_name, f_name):
+                discrete_function = DiscreteFunction(self.space, vec, vec_name)
+                discrete_function.visualize(f_name[:-4])
+
+            suffix = '.vtp' if self.space.dimDomain == 1 else '.vtu'
+            interactive = filename is None
+            to_clean_up = []
```

Comment:
Could you put the data into a temporary directory instead?

Reply:
If the user calls `visualize(U, filename='foo')`, he expects to find a `foo.vtu` in the CWD (and no interactive gui). If he calls `visualize(U)` he expects Paraview to open, see line 62ff. So I think it is fine the way it is, or am I missing something?

Reply:
> am I missing something

No, I think I misunderstood the intention here.  
Why would I always want to remove the files after they're viewed in paraview though? 
Currently opening paraview is blocking, right? The builtin Qt one makes that optional IIRC. Could be an option here too. 
The actual program to execute could  be a `@defaulted` configuration.

Reply:
Is it me or are the comments here missing from the discussion page? What do you mean by `The builtin Qt one`? If there is an alternative way to plot vtk data in an interactive (non jupyter notebook) session without Paraview, I am happy to support that.

Reply:
I think it's you? I type this from the main PR view. 
I mean the QT based visualizer for builtin discretizations.

Reply:
s.a.